### PR TITLE
Display current Total Supply on market modal

### PR DESCRIPTION
**Pull request by j923sneaks**
hello, `beastrong23` here from discord. Opening a PR for displaying latest SFF total supply

- call `cacheTotalSupply()` if opened
- render `Loading...` when value is 0

### CODE REVIEW

**Review by adamhannigan — APPROVED** [review]
Great work on your first PR. This will help reduce a lot of confusion.

I have tested and it updates the dynamic values after planting/harvesting 👏 

Feel free to merge this in. You have probably already noticed but this will kick off a Github Action that will automatically deploy your change to sunflower-farmers.com/play. You should see it pop up in a few minutes after merging. 🚀

**Comment by j923sneaks** [comment]
I don't think I have write access but it's fine w/ me if you merge this anytime :)

**Review by Rofrtd — APPROVED** [review]
Nice work mate. 
I've tested it and its working well. :shipit: 
I'll merge it in as you don't have write access. Thanks for the contribution! 🚀

**Pull request merged**
